src/store/ModalsModule.ts

```diff
@@ -0,0 +1,20 @@
+interface State {
+    isGameRunningModalOpen: boolean;
+}
+
+export default {
```

Comment:
Potentially nicer to be under `/src/store/modules/ModalsModule.ts

Reply:
Moved to subdir.